src/main/java/redis/clients/jedis/JedisSentinelPool.java

```diff
@@ -13,6 +13,10 @@
 import redis.clients.jedis.exceptions.JedisConnectionException;
 import redis.clients.jedis.exceptions.JedisException;
 
+import javax.net.ssl.HostnameVerifier;
+import javax.net.ssl.SSLParameters;
+import javax.net.ssl.SSLSocketFactory;
+
 public class JedisSentinelPool extends JedisPoolAbstract {
 
   protected GenericObjectPoolConfig poolConfig;
```

Comment:
`GenericObjectPoolConfig` is a raw type, how about sprinkling some `@SupressWarnings` if it cannot be parameterized with `<?>`

Reply:
@bsbodden That's 7 years old code. No one is touching that is working. You can take the dare yourself and touch it :D

Reply:
I'm making this resolved as the PR didn't modify it.